exp-models/addon/models/experiment.js

```diff
@@ -24,6 +24,15 @@ export default DS.Model.extend(JamModel, {
     endDate: DS.attr('date'),
     structure: DS.attr(),
 
+    thumbnailId: DS.attr('string'),
```

Comment:
Discussed in person with Sam. It seems the separate collection was a requirement to prevent elasticsearch from indexing this data. 

It might help to add a brief source code comment to this effect (to keep future maintainers from overzealous refactorings), but not essential.